docs/RationaleHW.md

```diff
@@ -39,21 +39,23 @@ several major contributions:
 
  1) The `hw` dialect provides unifying structure and
     abstractions that are useful for a wide range of hardware modeling problems.
-    Is allows other dialects to "mix in" with it to provide higher level
+    It allows other dialects to "mix in" with it to provide higher level
     functionality. `hw` is roughly akin to the "std" dialect in MLIR (but better
     curated).
  2) The `comb` dialect provides a common set of operations for combinational
     logic.  This dialect is designed to allow easy analysis and transformation.
  3) The `sv` dialect provides direct access to a wide variety
     of SystemVerilog constructs, including behavioral constructs, syntactic
     sugar constructs, and even idioms like `ifdef` blocks.
- 4) By providing a high quality implementation and a number of useful compiler
+ 4) The `circt` project provides a high quality implementation and a number of useful compiler
     passes for analyzing and transforming these dialects, and a SystemVerilog
     emitter that generates pretty output.
 
 The combination of these capabilities provides a useful suite of functionality
 for compiler tools that want to generate high quality SystemVerilog.
 
+_MW: Where is sequential logic handled (registers, memories)? Is that all lumped into `sv`, or...?_
```

Comment:
We have a `seq` dialect in early development, but it isn't really ready to recommend to folks who aren't actively developing it IMO.   It isn't used by the FIRRTL path.